lib/experimental/Navigation/Header/Breadcrumbs/index.tsx

```diff
@@ -218,7 +218,10 @@ const BreadcrumbContent = React.forwardRef<
     ),
     link: (
       <ShadBreadcrumbLink asChild className="p-0">
-        <Link {...("href" in item ? item : {})} className="block">
+        <Link
+          {...("href" in item ? (item as BreadcrumbNavItemType) : {})}
```

Comment:
Do we really need a type assertion here? They are usually dangerous, and it looks like `"href" in item` should already be able to discriminate, no?